engine/src/main/java/org/terasology/input/lwjgl/LwjglKeyboardDevice.java

```diff
@@ -15,40 +15,249 @@
  */
 package org.terasology.input.lwjgl;
 
-import com.google.common.collect.Queues;
-
-import org.lwjgl.input.Keyboard;
+import com.google.common.collect.Lists;
+import gnu.trove.map.TIntIntMap;
+import gnu.trove.map.hash.TIntIntHashMap;
+import gnu.trove.set.TIntSet;
+import gnu.trove.set.hash.TIntHashSet;
+import org.lwjgl.glfw.GLFW;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.terasology.input.ButtonState;
 import org.terasology.input.Input;
 import org.terasology.input.InputType;
+import org.terasology.input.Keyboard;
 import org.terasology.input.device.KeyboardAction;
 import org.terasology.input.device.KeyboardDevice;
 
+import java.util.Iterator;
 import java.util.Queue;
 
 /**
+ *
```

Comment:
Eeee - more of these old empty javadoc blocks. If you could fill them in with some basic javadoc then cool. If not and you see empty javadoc blocks please just remove them so the code analytics will complain about there being no javadoc